src/plots/gl3d/scene.js

```diff
@@ -58,10 +58,10 @@ function render(scene) {
     }
 
     function formatter(axisName, val) {
-        if(typeof val === 'string') return val;
-
         var axis = scene.fullSceneLayout[axisName];
-        return Axes.tickText(axis, axis.c2l(val), 'hover').text;
+        var calendar = trace[axis._id.charAt(0) + 'calendar'];
+
+        return Axes.tickText(axis, axis.d2l(val, 0, calendar), 'hover').text;
```

Comment:
This one here 

https://github.com/plotly/plotly.js/pull/1414/files#diff-c406dc5b6d20f7f95206516178b5be36R148

should do it.